### PR TITLE
[Laravel][Docs] Remove indentation

### PR DESCRIPTION
**Pull request by Nyholm**
It will render better in mobile

### CODE REVIEW

File: docs/integration/laravel.md [hunk 1]
```diff
@@ -27,15 +27,15 @@ key will not work.
...
-    'dynamodb' => [
--       'driver' => 'dynamodb',
-+       'driver' => 'async-aws-dynamodb',
-        'key' => env('AWS_ACCESS_KEY_ID'),
-        'secret' => env('AWS_SECRET_ACCESS_KEY'),
-        'region' => env('AWS_DEFAULT_REGION', 'us-east-1'),
-        'table' => env('DYNAMODB_CACHE_TABLE', 'cache'),
-        'endpoint' => env('DYNAMODB_ENDPOINT'),
-    ],
+'dynamodb' => [
```
Comment (jderusse) [inline docs/integration/laravel.md:30]: missing space  to avoid breaking `+`, `-` indentation

Comment (Nyholm) [reply docs/integration/laravel.md:30]: Actually no. It surprised me too. 

Screenshot 2020-05-10 at 11 39 13

Comment (jderusse) [reply docs/integration/laravel.md:30]: the `'` of `'dynamodb` should not be aligned with `-` . Same for `],`
when indenting diff, all lignes are prefixed with `-`, `+` or ` `

Comment (jderusse) [reply docs/integration/laravel.md:30]: example of issue
```diff
-'foo' => null,
+'Foo' => null,
'dynamodb' => [
-   'driver' => 'dynamodb',
+   'driver' => 'async-aws-dynamodb',
    'key' => env('AWS_ACCESS_KEY_ID'),
],
'Bar' => null,
-'baz' => null,
+'Baz' => null,

Comment (Nyholm) [reply docs/integration/laravel.md]: Ah, Okey. I understand now. I've fixed the diff and I've also modified other diffs with the same issue

**Comment by Nyholm** [comment]
Thank you.

**Pull request merged**